rfc-163-cdn-rationalisation.md

```diff
@@ -0,0 +1,39 @@
+# Simplify our CDN config, and standardise on Terraform for deployment
+
+## Summary
+
+- Where possible, relocate logic from the CDN layer to other parts of the stack
```

Comment:
Is this scoped just for the GOV.UK service in Fastly, or does it indicate where some of the others should go as well?

Reply:
Good point! So far I've only looked at the GOV.UK service, but I should probably expand the scope of this RFC to include all of them...